core/domain/email_jobs_one_off_test.py

```diff
@@ -45,6 +45,17 @@ def test_hashes_get_generated(self):
         # pylint: disable=unused-argument
         def _generate_hash_for_tests(
                 cls, recipient_id, email_subject, email_body):
+            """Generates hash for tests.
+            Args:
```

Comment:
Please add one blank line before `Args` and `Returns` section as it was in your previous PR. Ditto everywhere else.

Reply:
Ah OK! Didn't know that was a convention too..
Thanks!